docs/project/changelog.md

```diff
@@ -47,6 +47,12 @@ myst:
   streams that are passed to `loadPyodide()` or by `pyodide.setStdout()` and `pyodide.setStderr()`.
   {pr}`5621`
 
+- {{ Enhancement }} The `enableRunUntilComplete` option to `loadPyodide()` is
+  now on by default. This makes `run_until_complete` block using stack
+  switching, or crash if stack switching is disabled. If you need the old no-op
```

Comment:
If it makes `run_until_complete` crash, then I think it should be tagged as a breaking change.